build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -26,36 +26,41 @@
       <property name="ignoreFinal" value="false"/>
       <property name="tokens" value="ANNOTATION_FIELD_DEF,PARAMETER_DEF,VARIABLE_DEF,METHOD_DEF"/>
     </module>
+    <module name="AnnotationLocation">
+      <property name="allowSamelineMultipleAnnotations" value="false"/>
+      <property name="allowSamelineSingleParameterlessAnnotation" value="false"/>
+      <property name="allowSamelineParameterizedAnnotation" value="false"/>
+    </module>
     <module name="ArrayTypeStyle">
-      <property name="severity" value="error"/>
       <message key="array.type.style" value="Array must be defined as Foo[] array, not Foo array[]"/>
     </module>
-    <module name="AvoidStarImport">
-      <property name="severity" value="error"/>
+    <module name="AvoidStarImport"/>
+    <module name="CatchParameterName">
+      <property name="format" value="^(ex|e)$"/>
```

Comment:
Variables like `ex2` and`ignored` are also common, and fine.

Reply:
Added.